src/components/AnswerCard.js

```diff
@@ -0,0 +1,31 @@
+import { useCheckResult } from "../App";
+
+export default function AnswerCard({
+  answer,
+  onClick,
+  selectedAnswer,
+  correctAnswer,
+}) {
+  const { isChecked } = useCheckResult();
+
+  return (
+    <div
+      onClick={onClick}
+      className={`text-xl cursor-pointer ${
+        selectedAnswer === answer
```

Comment:
can you please use this package called [classnames](https://www.npmjs.com/package/classnames)  in order to make clean code